tests/utils/exec/idps.go

```diff
@@ -0,0 +1,104 @@
+package exec
+
+import (
+	"context"
+	"fmt"
+
+	CON "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	h "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/helper"
+)
+
+type IDPArgs struct {
+	ClusterID     string            `json:"cluster_id,omitempty"`
+	Name          string            `json:"name,omitempty"`
+	ID            string            `json:"id,omitempty"`
+	Token         string            `json:"token,omitempty"`
+	OCMENV        string            `json:"ocm_environment,omitempty"`
+	URL           string            `json:"url,omitempty"`
+	MappingMethod string            `json:"mapping_method,omitempty"`
+	GitHub        map[string]string `json:"github,omitempty"`
```

Comment:
thinking now if we need Github/gitlab/htpasswd/OpenID Args here if they are hardcoded in the manifest and for defining the Directory - we specify it in initializating..

Reply:
I agree, in any case we can later on bring them back if we will need
removed them for now